src/ansys/fluent/core/quantity.py

```diff
@@ -333,6 +338,12 @@ def _compute_multipliers(self, unit_str, power):
                     _UnitsTable.derived_units[unit_str], term_power
                 )
 
+    def _compute_offset(self, unit_str):
+        if unit_str == "C" or unit_str == "C^1":
+            self._si_offset = 273.15
```

Comment:
I don't see any changes where we are removing `273.15` (nor other numbers essentially derived from that offset), so we need to keep in mind the need to refactor subsequently.

Reply:
Yes, we will definitely refactor all code. thank you. Right now code is updated and passing all test cases. Thank you.